### PR TITLE
opae-sdk: add mock sysfs c6100 to test framework

### PR DESCRIPTION
**Pull request by anandaravuri**
Signed-off-by: anandaravuri <ananda.ravuri@intel.com>

### CODE REVIEW

File: tests/framework/platform/fpga_hw.cpp [hunk 1]
```diff
@@ -187,6 +187,25 @@ R"mdata({"version": 1,
...
+     "accelerator-clusters":
+      [
+        {
+          "total-contexts": 1,
+          "name": "nlb0",
+          "accelerator-type-uuid": "9aeffe5f-8457-0612-c000-c9660d824272"
+        }
+      ]
+     },
+     "platform-name": "PAC"}";
```
Comment (tswhison) [inline tests/framework/platform/fpga_hw.cpp:206]: Remove ;

**Review by tswhison — COMMENTED** [review]
Did you use tests/framework/create-mock.py to create the mock tarball?

File: tests/opae-c/test_pluginmgr_c.cpp [hunk 2]
```diff
@@ -486,6 +486,7 @@ TEST_P(pluginmgr_c_p, fpgaReset_null_handle) {
   opae_plugin_mgr_finalize_all();
 }
 
+GTEST_ALLOW_UNINSTANTIATED_PARAMETERIZED_TEST(pluginxtest_c_p);
```
Comment (tswhison) [inline tests/opae-c/test_pluginmgr_c.cpp:489]: Shouldn't this be pluginmgr_c, instead of pluginxtest_c_p?

Comment (anandaravuri) [reply tests/opae-c/test_pluginmgr_c.cpp]: fixed

File: tests/xfpga/test_sysfs_c.cpp [hunk 3]
```diff
@@ -292,9 +264,51 @@ INSTANTIATE_TEST_SUITE_P(sysfsinit_c, sysfsinit_c_p,
                                                                         "dfl-d5005",
                                                                         "dfl-n3000",
                                                                         "dfl-n6000-sku0",
-                                                                        "dfl-n6000-sku1"
+                                                                        "dfl-n6000-sku1",
+                                                                        "dfl-c6100"
                                                                       })));
 
+class sysfsinit_bbsid_c_p :public  sysfsinit_c_p
```
Comment (tswhison) [inline tests/xfpga/test_sysfs_c.cpp:271]: Just curious, why did this need to be moved?

Comment (anandaravuri) [reply tests/xfpga/test_sysfs_c.cpp:271]: C6100 doesn't support PR, get_interface_id is not valid test cases
get_interface_id test case works for N6000, D5005 pcie cards.

**Review by tswhison — APPROVED** [review]

**Review by michelleyho — APPROVED** [review]

**Pull request merged**